pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -115,10 +115,14 @@ def my_func(self):
             """This is a docstring.
 
             :raises RuntimeError: Always
-            :raises NameError: Never
+            :except NameError: Never
```

Comment:
Can you also add a negative test, where we are using one or two of the alternative names, but inside the function, we are not raising them at all?